README.md

```diff
@@ -1,79 +1,136 @@
+[htsget-diagram-png]: https://samtools.github.io/hts-specs/pub/htsget-ticket.png
+[tokio]: https://github.com/tokio-rs/tokio
+
+## Usage
 
+Htsget-rs is configured using environment variables, for details on how to set them, see [htsget-config].
+
+### Local
+To run a local instance htsget-rs, run [htsget-http-actix] by executing the following:
+```sh
+cargo run -p htsget-http-actix
 ```
-$ cargo run -p htsget-http-actix
+Using the default configuration, this will start a ticket server on `127.0.0.1:8080` and a data block server on `127.0.0.1:8081`
+with data accessible from the [data] directory. See [htsget-http-actix] for more information.
+
+### Cloud
+Cloud based htsget-rs uses [htsget-http-lambda]. For more information and an example deployment of this crate see 
+[deploy].
+
+### Tests
+
+Tests can be run tests by executing:
+
+```sh
+cargo test --all-features
 ```
 
-This uses the default settings with example files from the `data` directory and the server listening to requests on port 8080. Please refer to the [htsget-http-actix crate README.md for further details][htsget-http-actix-readme].
+To run benchmarks, see the benchmark sections of [htsget-http-actix][htsget-http-actix-benches] and [htsget-search][htsget-search-benches].
 
-### Cloud
+[htsget-http-actix-benches]: htsget-http-actix/README.md#Benchmarks
+[htsget-search-benches]: htsget-search/README.md#Benchmarks
 
-To deploy to an AWS account, please refer to the `deploy/README.md` for further instructions.
+## Project Layout
 
-## Intro
+This repository consists of a workspace composed of the following crates:
 
-Htsget makes bioinformatic data formats accessible through HTTP in a consistent way.
+- [htsget-config]: Configuration of the server.
+- [htsget-http-actix]: Local instance of the htsget server. Contains framework dependent code using [Actix Web][actix-web].
+- [htsget-http-core]: Handling of htsget HTTP requests. Framework independent code.
+- [htsget-http-lambda]: Cloud based instance of the htsget server. Contains framework dependent
+code using the [Rust Runtime for AWS Lambda][aws-lambda-rust-runtime].
+- [htsget-search]: Core logic needed to search bioinformatics files based on htsget queries.
+- [htsget-test-utils]: Test utilities used by other crates in the project.
 
-This repo implements a 100% Rust implementation of the [htsget spec][htsget-spec] using [Noodles][noodles] which gets rid of the [`unsafe` interfacing][rust-htslib] with the C-based [htslib](https://github.com/samtools/htslib).
+Other directories contain further applications or data:
+- [data]: Contains example data files which can be used by htsget-rs, in folders denoting the file type.
+This directory also contains example events used by a cloud instance of htsget-rs in the [`events`][data-events] subdirectory.
+- [deploy]: An example deployment of [htsget-http-lambda].
 
-Our Rust implementation distinguishes itself from others in the following ways:
```

Comment:
Nice new summary. I think it's also absolutely okay to highlight features of htsget-rs here (mention that it supports BAM/CRAM/VCF, that it's serverless, comes with benchmarks). Totally okay to advertise features that people are looking for!

Reply:
Thanks for reading over it! Yeah, agreed. I've added a bit more commentary about the features in the intro paragraphs.